src/Tools/ExternalAccess/OmniSharp/Structure/OmniSharpBlockTypes.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using Microsoft.CodeAnalysis.Structure;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.OmniSharp.Structure
+{
+    internal static class OmniSharpBlockTypes
+    {
+        // Basic types.
+        public static string Nonstructural => BlockTypes.Nonstructural;
```

Comment:
Do you want any sort of test that these match? Your enum test was nice but not sure it's easy to make it work for this too.

Reply:
I tried to think of a way to do it neatly, but it would just be a bunch of reflection over the properties and I didn't feel it was worth it.